Assets/_BForBoss/_Weapons/Prefabs/RifleBullet_NoPhysics.prefab

```diff
@@ -0,0 +1,79 @@
+%YAML 1.1
```

Comment:
Look through the prefabs, wondering why there's a cylinder and a sphere mesh on this.

Reply:
![image](https://user-images.githubusercontent.com/31762637/144982575-3e4d4696-01b9-425f-aa1c-807463dc922d.png)
I figured this looked like a decent placeholder bullet